app/javascript/src/Login/components/LoginPage.tsx

```diff
@@ -56,11 +56,9 @@ const LoginPage: FunctionComponent<Props> = ({
         session={session}
       />
     )}
-    {authenticationProviders && (
-      <div className="providers-separator">
-        <AuthenticationProviders authenticationProviders={authenticationProviders} />
-      </div>
-    )}
+    <div className="providers-separator">
```

Comment:
Before this PR, `authenticationProviders` was optional. Now it's not optional therefore we don't need the conditional rendering.

Reply:
well, yeah, but why we are making it not optional? Well, OK, we can say - it's  never null, but it can be empty.

But I think it's reasonable to not render providers-related divs if the array is empty.

It's just a nitpick (not visible, just some `div`s in the source), and I am just trying to understand the reason behind it :grimacing:

Reply:
There's been no change just a bad type definition. `authenticationProviders` was never optional (it detauls to `[]`) therefore `AuthenticationProviders` is always rendered. It's just a simplification.

Reply:
> But I think it's reasonable to not render providers-related divs if the array is empty.

That can be done 🫡